go.mod

```diff
@@ -125,8 +127,7 @@ replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 	github.com/aws/aws-sdk-go v1.25.48 => github.com/aws/aws-sdk-go v1.33.0
 	github.com/aws/aws-sdk-go v1.27.0 => github.com/aws/aws-sdk-go v1.33.0
```

Comment:
Since you're targeting only one version, you should remove the version number from one of these replacements, and then specify only v1.33.0

If you actually need several different versions then you should refactor upstream

Reply:
Same as above. I am retargeting two different package version requests from two indirect dependencies to fix CVE issues